yarn-project/prover-client/src/orchestrator/orchestrator_lifecycle.test.ts

```diff
@@ -80,65 +62,71 @@ describe('prover/orchestrator', () => {
         expect((result1 as ProvingFailure).reason).toBe('Proving cancelled');
       }
 
-      await builderDb.rollback();
+      await context.actualDb.rollback();
```

Comment:
Unrelated to the current changes, but shouldn't `rollback` be tied to `cancelBlock`?